HACKING.md

```diff
@@ -223,3 +222,7 @@ Ex)
 ### Log level
 
 To set the log level of Strimzi for system tests need to add system property `TEST_STRIMZI_LOG_LEVEL` with one of the following values: `ERROR`, `WARNING`, `INFO`, `DEBUG`, `TRACE`.
+
+### Docker image names in ST
+
+To set specific prefix or suffix to the image name need to add system property `DOCKER_IMAGE_PREFIX` or `DOCKER_IMAGE_SUFFIX`
```

Comment:
Could you explain on some example what do `DOCKER_IMAGE_PREFIX` and `DOCKER_IMAGE_SUFFIX` actually mean / do?